### ISSUE TITLE
Pathway: acyl-CoA metabolic process

### ISSUE THREAD

**Issue by ValWood** [pombase/mitochondrion#5]
http://noctua.geneontology.org/workbench/noctua-visual-pathway-editor/?model_id=gomodel%3A67086be200000251
broaden this pathway to "acyl-CoA metabolic process"

https://www.pombase.org/results/from/id/83cb2ae5-5821-42df-8771-18f5533d2ca7

16/32 are mitochondrial

You need to follow
https://github.com/geneontology/go-ontology/issues/24350
see specifically https://github.com/geneontology/go-ontology/issues/24350#issuecomment-2416754175
and
check out [micropub-biology-001209.pdf](https://github.com/user-attachments/files/17395899/micropub-biology-001209.pdf)

**Comment by ValWood** [comment]
I had a note for "acetyl-CoA biosynthetic process from pyruvate" that I need to find a source of the  "mitochondria Matrix annotations, currently only have Mitochondrion"

**Comment by ValWood** [comment]
When an annotation is missing, I usually find an experimental source I  can ISO to , or some review to TAS, or make an IC (for example if the MF is known to be in a specific location). It would be  more useful if we find an EXP and ask Marc to PAINT it (I know Steven does this sometimes).

This will happen quite a lot during this project as we try to refine locations.

I use Helens tip to see if there are any existing EXP for any orthologs at the Alliance (you can retrieve all EXP annotations for orthologs)

For acetyl-CoA biosynthetic process from pyruvate, 

I just checked  pda1 (using human PDHA). SGD is annotated to nucleoid  (which is part of matrix), so Marc might be able to PAINT based on this.

I suggest we link the PAINT tickets to these pathway tickets so that everyone can see that the request is in progress.
@PCarme  I can show you this on Monday or Tuesday if unclear.

**Comment by ValWood** [comment]
PAINT REQUESTS

| gene |  PANTHER  |  term | ticket
| --- | --- | ---- | ---- |
| S.p  pda1 (human PDHA) | PTN008519424  |  mitochondrial matrix | https://github.com/geneontology/go-annotation/issues/5485 |
| - | - | - | - |

**Comment by PCarme** [comment]
We should add [pyruvate dehydrogenase (acetyl-transferring)]-phosphatase activity(GO:0004741) to https://www.pombase.org/gene/SPAC4A8.03c, based on ISO with https://www.yeastgenome.org/locus/S000002133

Note: the other PDHC phosphatase subunit, https://www.pombase.org/gene/SPAC10F6.17c, is annotated to this term with an IBA evidence.

**Comment by PCarme** [comment]
Also noticing that https://www.pombase.org/gene/SPCC794.07 has his gene product described as "e3 component", while his orthologs https://www.yeastgenome.org/locus/S000005015 https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:2896 are described as the E2 of PDHC. 
https://www.pombase.org/gene/SPAC1002.09c is the E3.

**Comment by PCarme** [comment]
In the "acetyl-CoA biosynthetic process from panthotenate, need to add ptk1, hal3, cab4 and cab5 (IC with their respective molecular functions or with "CoA biosynthetic process"

**Comment by PCarme** [comment]
Also, should fas1 and fas2 be annotated to "long-chain fatty-acyl-CoA biosynthetic process" (GO:0035338) based on their MF annotation to https://www.pombase.org/term/GO:0004321

**Comment by ValWood** [comment]
>Also, should fas1 and fas2 be annotated to "long-chain fatty-acyl-CoA biosynthetic process" (GO:0035338) based on their MF annotation to https://www.pombase.org/term/GO:0004321

Yes these could be annotated experimentally

**Comment by ValWood** [comment]
> Also noticing that https://www.pombase.org/gene/SPCC794.07 has his gene product described as "e3 component", while his orthologs https://www.yeastgenome.org/locus/S000005015 https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:2896 are described as the E2 of PDHC.

yep, I think I introduced that error recently when trying to standardize the case! definitely  fix

**Comment by ValWood** [comment]
> We should add [pyruvate dehydrogenase (acetyl-transferring)]-phosphatase activity(GO:0004741) to https://www.pombase.org/gene/SPAC4A8.03c, based on ISO with https://www.yeastgenome.org/locus/S000002133

It looks like it but I'm not sure why this has its own gene product-specific GO term @pgaudet  should this term exist? is it special in some metabolic way?

**Comment by sjm41** [comment]
Just to note that GO also has a term for the corresponding kinase:
pyruvate dehydrogenase (acetyl-transferring) kinase activity | GO:0004740

GO has several terms for specific kinases, less terms for specific phosphatases.
This may just be a case of GO following EC's lead?

**Comment by pgaudet** [comment]
> GO has several terms for specific kinases, less terms for specific phosphatases.
This may just be a case of GO following EC's lead?

Yes
https://enzyme.expasy.org/EC/2.7.11.-

We can decide to obsolete these, but then people should include inputs in their annotations. 
Right now there are no inputs in the logical definitions of these terms, as we dont have a species-agnostic entity we can link in the terms. Maybe a good solution is using a PTN ID? 

Pascale

**Comment by hattrill** [comment]
Adding "pyruvate decarboxylation to acetyl-CoA" which feeds into TCA

pyruvate decarboxylation to acetyl-CoA (Dmel) [VPE](http://noctua.geneontology.org/workbench/noctua-visual-pathway-editor/?model_id=gomodel:67e5e74400002207) / [Pathway View](http://noctua.geneontology.org/workbench/noctua-alliance-pathway-preview/?model_id=gomodel:67e5e74400002207)

pyruvate decarboxylation to acetyl-CoA - alternative version for complexes (Dmel) [VPE](http://noctua.geneontology.org/workbench/noctua-alliance-pathway-preview/?model_id=gomodel:67f85f2b00001548)) / [Pathway View](http://noctua.geneontology.org/workbench/noctua-visual-pathway-editor/?model_id=gomodel:67f85f2b00001548)

This feeds into the TCA #25
